docs/rewards/rewards-reference.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 3
+---
+
+# Rewards Reference 
+This guide is a list of the availabe queries and commands related to the rewards system.
+
+## Rewards Queries 
+
+``archwayd query rewards block-rewards-tracking`` - Query rewards tracking data for the current block height
+
+``archwayd query rewards contract-metadata`` - Query contract metadata (contract rewards parameters)
+
+``archwayd query rewards estimate-fees`` - Query transaction fees estimation for a given gas limit
+
+``archwayd query rewards outstanding-rewards``-  Query current credited rewards for a given address (the address set in contract(s) metadata rewards_address field)
+
+``archwayd query rewards params`` - Query module parameters
+
+``archwayd query rewards pool`` - Query the undistributed rewards pool (ready for withdrawal) and the treasury pool funds
+
+``archwayd query rewards rewards-records`` - Query rewards records stored for a given address (the address set in contract(s) metadata rewards_address field) with pagination 
+
+## Transaction Commands 
+
+``archwayd tx set-contract-metadata`` -  Create / modify contract metadata (contract rewards parameters)
+
+``archwayd tx withdraw-rewards`` - Withdraw current credited rewards for the transaction sender
```

Comment:
also, i cannot find the `set-contract-metadata` and `withdraw-rewards` commands, using archwayd `0.0.5`, as i get:

`Error: unknown command "set-contract-metadata" for "tx"`
or
`Error: unknown command "withdraw-rewards" for "tx"`

Reply:
@aelesbao are you aware if the issue i am encountering is due to a specific binary version i should use?